.clj-kondo/com.github.seancorfield/next.jdbc/config.edn

```diff
@@ -1,5 +1,8 @@
 {:hooks
  {:analyze-call
   {next.jdbc/with-transaction
-   hooks.com.github.seancorfield.next-jdbc/with-transaction}}
```

Comment:
I ran Kondo locally and these are the new imports. Not strictly related to this PR but we want to check these in so I'm doing it here